src/UsgsAstroLsSensorModel.cpp

```diff
@@ -2138,19 +2138,6 @@ void UsgsAstroLsSensorModel::losEllipsoidIntersect(
    computeElevation(x, y, z, h, aPrec, desired_precision);
    slope = -1;
 
-   while (MKTR > ktr && fabs(height - h) > desired_precision)
-   {
-      sprev = scale;
-      scale += slope * (height - h);
-      x = xc + scale * xl;
-      y = yc + scale * yl;
-      z = zc + scale * zl;
-      hprev = h;
-      computeElevation(x, y, z, h, aPrec, desired_precision);
-      slope = (sprev - scale) / (hprev - h);
-      ktr++;
-   }
-

```

Comment:
I'm not sure why this code block is here, but it seems to be an odd iterative approach that is already handled based on the math we do earlier.

Reply:
Is this a hold over form the original author? If height = 0 this intersection should always be computable in a single step? Vector intersection on the ellipsoid shouldn't need an iterative solution, but what if the height provided is above or below the ellipsoid? Is this handling occlusion where the ellipsoid can occlude itself because the provided height is 'under' the surface?

Reply:
You should be able to just add height to the ellipsoid radii before you do the intersection. That is also solvable analytically.

Reply:
I don't think we're doing this right now. I may add that in under this PR.

Reply:
It turns out height is already being added to the radii, so this is ready to go now.